book/navigation.rst

```diff
@@ -0,0 +1,118 @@
+Managing the Application Menu
+=============================
+
+Both the Oro Platform application and the OroCRM application come with a rich user interface. Each
+part of the application can be accessed by browsing the application using the provided navigation
+items.
+
+The Oro Platform leverages the famous `KnpMenuBundle`_ to provide highly customizable menus. You
+can add your own menu items to access your project specific interfaces or even replace existing
+items.
+
+Mastering the application menu is a two-step process:
+
+#. :ref:`Create the new navigation items <book-navigation-create-menu-item>`
+#. :ref:`Compose trees of navigation items <book-navigation-compose-tree>`
+
+The `OroNavigationBundle`_ automatically processes a YAML configuration file which is named
+``navigation.yml`` when it is placed in the ``Resources/config`` directory of a registered bundle.
+The menu configuration needs to be placed under the ``oro_menu_config`` tree.
+
+.. _book-navigation-create-menu-item:
+
+Creating Menu Items
+-------------------
+
+You can create new navigation under the ``items`` key. Each item must be identified by a unique
+name which acts as a key in the menu configuration:
+
+.. code-block:: yaml
+    :linenos:
+
+    # src/Acme/DemoBundle/Resources/config/navigation.yml
+    oro_menu_config:
+        items:
+            blog:
+                label: acme_demo.menu.blog
+                uri: '#'
+            blog_categories:
+                label: acme_demo.menu.blog_categories
+                route: acme_demo.blog_categories
+            blog_index:
+                label: acme_demo.menu.blog_overview
+                route: acme_demo.blog_index
+
+The example above defines three menu items:
+
+* The ``blog`` item consists of a label and the URI ``#``. This means that the item will not react
+  on mouse clicks, but can be used as a placeholder for nested menus.
+
+* Both the ``blog_categories`` and the ``blog_index`` items each reference a an existing route.
```

Comment:
Should be  

Both the ``blog_categories`` and the ``blog_index`` items reference an existing route. 

or 

Each of the ``blog_categories`` and the ``blog_index`` items references an existing route.